### PR TITLE
Update Brexit landing page

### PR DESCRIPTION
**Pull request by alex-ju**
### What & Why
Update Brexit landing to match [new design](https://brexit-landing.herokuapp.com/landing-sections).

### Notes
 - This PR is missing the Welsh translation (to be added when available).
 - ~~The changes in this PR contain a feature that has [not yet been released (`font_size` on action link component)](https://github.com/alphagov/govuk_publishing_components/pull/2162)~~.

### Visual changes
<table>
<tr><th>Before</th><th>After</th></tr>
<tr><td valign="top">

brexit_landing_page_before

</td><td valign="top">

brexit_landing_page_after

</td></tr>
</table>

[Trello card](https://trello.com/c/psZUFhNw)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/presenters/brexit_landing_page_presenter.rb [hunk 1]
```diff
@@ -12,7 +12,6 @@ class BrexitLandingPagePresenter
 
   def initialize(taxon)
     @taxon = taxon
-    @comms = fetch_comms
```
Comment (hannako) [inline app/presenters/brexit_landing_page_presenter.rb:15]: We can remove the `:comms` attr_reader now

File: app/assets/stylesheets/components/_topic-list.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .app-c-topic-list__item {
-  @include govuk-font(19, $weight: bold);
+  @include govuk-font(19, $weight: bold, $line-height: 1.7);
```
Comment (hannako) [inline app/assets/stylesheets/components/_topic-list.scss:2]: should 1.7 be stored in a variable or does it not matter?

File: app/views/brexit_landing_page/_page_header.html.erb [hunk 3]
```diff
@@ -46,6 +46,12 @@
         dark_icon: true,
         margin_bottom: 4,
         font_size: "m",
+        data: {
+          "module": "gem-track-click",
+          "track-category": "brexit-landing-page",
+          "track-action": t("brexit_landing_page.page_header_links.guidance_for_businesses.path"),
+          "track-label": t("brexit_landing_page.page_header_links.guidance_for_businesses.text")
+        }
```
Comment (hannako) [inline app/views/brexit_landing_page/_page_header.html.erb:54]: very minor comment, but lines 52 and 53 are a bit long. We could use a variable to store the track-action and track-label values like we do [here](https://github.com/alphagov/collections/blob/62fad20d8fad6571e323e791c433e065e7540c60/app/views/brexit_landing_page/_page_header.html.erb#L51-L52)

**Review by hannako — COMMENTED** [review]
Looks great!! 

A few very minor comments. And wanted to check the survey link - is this how it normally looks on mobile?

Screenshot 2021-06-24 at 17 29 32

File: app/assets/stylesheets/components/_topic-list.scss (hunk 2 above)
Comment (alex-ju) [reply app/assets/stylesheets/components/_topic-list.scss:2]: Unless we use the value in multiple places, we only require variables when assigned directly as a property value – when passed as function arguments it's not required.

File: app/presenters/brexit_landing_page_presenter.rb (hunk 1 above)
Comment (alex-ju) [reply app/presenters/brexit_landing_page_presenter.rb:15]: Good spot! Updated.

**Comment by alex-ju** [comment]
> Looks great!!
> 
> A few very minor comments. And wanted to check the survey link - is this how it normally looks on mobile?
> 
> Screenshot 2021-06-24 at 17 29 32

Thanks for the swift review @hannako! I have raised [a separate PR in static](https://github.com/alphagov/static/pull/2530) to address the survey banner issue.

**Review by hannako — APPROVED** [review]
🎉

**Comment by alex-ju** [comment]
Rebased to main to get the latest GOV.UK Publishing Components release in.

**Pull request merged**